### DISCUSSION TITLE
Allow email rules based on individual apps/stores

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#4564] [Ideas / Feature Requests]
On my store account I have 5 different little stores. Which are called "Apps"

![MyApp](https://user-images.githubusercontent.com/31393558/214387251-c3c3ff08-f37f-42d3-95c3-43e9bd562409.jpg)

I setup an email notification so when a user creates and pays an invoice. I receive both emails. 

The issue is the email doesn't tell me which store/app the invoice is from.

I can tell which store/app it is from because of the price. I just have to make sure none of the stores/apps have the same price.

I suggest a drop down to choose which app.
![EmailRules](https://user-images.githubusercontent.com/31393558/214388479-38e6edf4-9b82-430f-a838-5807df4b4228.jpg)

**Comment by bolatovumar** [comment]
@git-3000 you can get around this issue by creating separate stores for each app and then you can set up separate rules for each one and can even include the store or app name in the "Subject" or "Body" fields.

Additionally, you can include any of the below tokens in these fields and they will be replaced with actual values when the email is sent out (e.g., `{Invoice.Currency}` will be replaced with the actual currency label such as `BTC`).

![Capture](https://user-images.githubusercontent.com/1934678/221472609-c18265ec-1092-4ff7-80dd-2cae1baf5935.PNG)
